### PR TITLE
feat: [CDS-68136]:pdc-connectore-fixed-issue-with-max-area-length

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Fixed issue with max length (1000 characterss.) for input area in PDC connector.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/35-connectors/components/CreateConnector/PdcConnector/components/UploadJSON.tsx [hunk 1]
```diff
@@ -23,9 +23,10 @@ interface UploadJSONInterface {
     selectionType: SelectionType
   }>
   previousHosts?: uploadHostItem[] | string
+  maxLength?: number
 }
 
-const UploadJSON = ({ setJsonValue, formikProps, previousHosts }: UploadJSONInterface) => {
+const UploadJSON = ({ setJsonValue, formikProps, previousHosts, maxLength = 100000 }: UploadJSONInterface) => {
```
Comment (PratyushGar) [inline src/modules/35-connectors/components/CreateConnector/PdcConnector/components/UploadJSON.tsx:29]: can we export and use the same variable for single source of truth

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger jest

**Comment by litvinovstas** [comment]
retrigger jest

**Comment by litvinovstas** [comment]
retrigger cypress-cd

**Comment by litvinovstas** [comment]
retrigger cypress-cd

**Review by PratyushGar — APPROVED** [review]

**Review by avinashmadhwani02 — APPROVED** [review]

**Pull request merged**